ToolBox/CoinApi/CoinApiSymbolMapper.cs

```diff
@@ -199,7 +199,10 @@ private void LoadSymbolMap(string[] exchangeIds)
             // <Exchange>_SPOT_<BaseCurrency>_<QuoteCurrency>_<ExtraSuffix>
             // Those cases should be ignored for SPOT prices.
             _symbolMap = result
-                .Where(x => x.IsActive && x.SymbolType == "SPOT" && x.SymbolId.Split('_').Length == 4)
+                .Where(x => x.SymbolType == "SPOT" &&
+                    x.SymbolId.Split('_').Length == 4 &&
+                    // exclude Bitfinex BCH pre-2018-fork as for now we don't have historical mapping data
+                    (x.ExchangeId != "BITFINEX" || x.AssetIdBase != "BCH" && x.AssetIdQuote != "BCH"))
```

Comment:
Was wondering what's the effect of excluding `BCH` here?

Reply:
The CoinApi tickers for BCH currently in the symbol list are:
- BCH - Bitcoin Cash before the Nov2018 fork (not active)
- BCHABC - Bitcoin Cash ABC (renamed) (active)
- BCHSV - Bitcoin Cash SV (fork) (active)

The Bitfinex tickers for BCH are:
- BAB - Bitcoin Cash ABC
- BSV - Bitcoin Cash SV

The Lean tickers for BCH ("bitfinex" market) are:
- BCH - Bitcoin Cash ABC
- BSV - Bitcoin Cash SV

This BCH exclusion is required after this missing mapping was added:
https://github.com/QuantConnect/Lean/pull/4888/files#diff-2cb47206de766f5b1077db5bb591cd81b51c350e6a52a8630570da460c3a2f9bR60